.github/workflows/release.yml

```diff
@@ -6,8 +6,10 @@
 name: release
 
 on:
+  repository_dispatch:
+    types: update
   schedule:
-    - cron:  '*/5 * * * *'
+    - cron:  '0 * * * *'
```

Comment:
We keep it running on every hour just in case the release process fails to dispatch the webhook. I am creating a PR on cli to add the webhook invocation.